### ISSUE TITLE
TypeError: n.includes is not a function - Firefox

### ISSUE THREAD

**Issue by singhpratik17** [liftoff/HumanInput#14]
When any of the fn+f2 or fn+f3 or fn+f4 combinations is pressed in firefox, I am getting an error:
TypeError: n.includes is not a function  - hi.min.js.

I fixed this issue temporarily by using n.hasOwnProperty instead of n.includes. Is there a better solution available?

**Comment by liftoff** [comment]
When you say, "fn+f2" I assume you mean you're just pressing the F2 key?  Well, let me clarify:  The Fn key on your keyboard doesn't actually register any key events on its own it merely changes what other keys do.  So on some keyboards Fn-F2 will generate, say a Play/Pause key event while on others it will just generate plain ol' F2.

Can you give me the OS type and version of Firefox this occurs with?  Also, does the same thing happen with Chrome?  It's probably an issue with Firefox or your OS.

Why it might be happening: If your OS doesn't generate the correct key event the browser won't generate a complete KeyEvent.  Rather, it will be an incomplete KeyEvent that doesn't have a 'code' property which would explain why the error is "n.includes is not a function"; because 'n' doesn't exist or has something that doesn't work with the includes() function (like if it's set to null).

I'll see if I can add some code to detect that sort of situation so it generates a better error message instead of an exception in the JS console but ultimately there's probably nothing I can do to fix those keys since it's probably an issue with the browser or the OS itself.

**Comment by liftoff** [comment]
Since I haven't heard back from you on this one I'm going to close it.  No one else seems to be having this issue so I'm going to assume my little edge case fix (in b1c8a1c) worked or that it was just something unique to your situation.

**Comment by drschwabe** [comment]
Same error here but only happens when I try to dispatch a native event (while an HI event is happening)

```
Uncaught TypeError: 
n.includes is not a function
```

the code to re-create: 
```
  //native keyboard event to trigger while ctrl-v isDown: 
  var e = new KeyboardEvent('keydown',{'keyCode':39,'which':39})

  if( HI.isDown('ctrl-v') ) {
      document.dispatchEvent(e) 
  }
```
